### PR TITLE
Validate Contact Form

### PR DESCRIPTION
**Pull request by CarlosIgreda**
## Portfolio Milestone: Validate Contact Form

For this milestone I:

✅ Used Gitflow
✅ Used linters
✅ Used basic JavaScript syntax
✅ Used JavaScript to manipulate DOM elements
✅ Used JavaScript events
✅ Used objects to store and access data
✅ Processed user input according to business rules
✅ Used client-side validation to catch and throw errors in the UI

### CODE REVIEW

File: interaction.js [hunk 1]
```diff
@@ -462,3 +463,31 @@ for (let i = 0; i < xicon.length; i++) {
...
+}
+
+document.addEventListener('submit', (e) => {
+  if (e.target.matches('form')) {
+    if (validation()) {
+      console.log('Form submitted successfully!');
+    } else {
+      e.preventDefault();
+      alert('Please enter your email in lower case');
+      $email.value = $email.value.toLowerCase();
```
Comment (AmaduKamara) [inline interaction.js]: - [x] From the lessons previously given to you, it's required that we follow [JavaScript Best Practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) and avoid using alert() and others you will find in that link. Kindly display an error text message close to the button when there is an uppercase letter in the email just as the requirement instructed and not alert(). 👍🏽

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️ 

Hi @CarlosIgreda,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!z

## Highlights
✅ You used Gitflow correctly 👍🏽
✅ Linter checks passed 👍🏽
✅ Validation check is implemented as expected for email 👍🏽

## Required Changes ♻️
_Check the comments under the review._

- [x] Kindly display the error message you wrote in the alert() in a p tag close to the Submit button or around the input field just as the requirement instructed 👍🏽

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: interaction.js (hunk 1 above)
Comment (CarlosIgreda) [reply interaction.js]: Done @AmaduKamara! Thanks for your feedback. I've already made the changes.

**Review by pinkmoon25 — APPROVED** [review]
Hi @CarlosIgreda & @LarryIVC , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Linters :heavy_check_mark: 
Gitflow :heavy_check_mark: 
Lowercase Validation :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by CarlosIgreda** [comment]
Thanks for your code review and feedback @PinkMoon25! I'm doing my best.

**Pull request merged**